### PR TITLE
docs(dev): improve doc for developers with mkdocs

### PR DESCRIPTION
**Pull request by lrubiorod**
This PR add instructions to install rust and mkdocs, and clang requirement.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -5,13 +5,25 @@ Witnet-rust is build using [The Rust Programming language](https://rust-lang-nur
 
 1. Install dependencies
     - Rust 1.31 (currently on the `nightly` release channel)
+    ```
+    $ curl https://sh.rustup.rs -sSf | sh
+    $ rustup default nightly
+    $ rustc --version
+    ```
+    - `clang` Clang compiler
```
Comment (mariocao) [inline docs/development.md]: :1st_place_medal:

**Comment by mariocao** [comment]
I would also update some instructions of the Development guide that are outdated regarding the `server` component. I think that right now we are calling it `node`.

For that reason, step 5 should be:
```mkdocs
$ RUST_LOG=witnet=trace cargo run node
```

Additionally, we should also update the `Justfile` and its documentation (also in `development.md`) accordingly,.

**Comment by lrubiorod** [comment]
> I would also update some instructions of the Development guide that are outdated regarding the `server` component. I think that right now we are calling it `node`.
> 
> For that reason, step 5 should be:
> 
> ```
> $ RUST_LOG=witnet=trace cargo run node
> ```
> 
> Additionally, we should also update the `Justfile` and its documentation (also in `development.md`) accordingly,.

It is fixed in PR #80

**Comment by mariocao** [comment]
This PR is superseded by #80

**Pull request closed without merging**